Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -346,6 +351,26 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        self.preferenceHelper.universalLinkUrl = [userActivity.webpageURL absoluteString];
```

Comment:
yeah, you're right. we still need to figure out what we're going to do here, we've been discussing alternatives today. We'll probably address next week.